### DISCUSSION TITLE
提示文件已损坏

### DISCUSSION THREAD

**Discussion by liuhao320** [nextai-translator/nextai-translator#321] [Q&A]
下载后打开就提示已损坏怎么办呢
![SCR-20230316-jkha](https://user-images.githubusercontent.com/38712559/225492726-646c2be5-dabe-4a9b-9eed-caae05f48be5.png)

**Comment by boldmatt** [comment]
我从24版开始遇到同样问题

**Comment by peteisacat** [comment]
作者在安装说明里写了解决方案。

**Reply by boldmatt** [reply]
找了半天没看到解决方案

**Reply by peteisacat** [reply]
https://blog.csdn.net/qq_32526375/article/details/127236507

**Comment by okisdev** [comment]
@liuhao320 请看 [故障排除](https://github.com/yetone/openai-translator/blob/main/README-CN.md#%E6%95%85%E9%9A%9C%E6%8E%92%E9%99%A4)
